game/menu/README.md

```diff
@@ -0,0 +1,49 @@
+# Infos for making Minigames
```

Comment:
Why is this in menu if it is for minigames though? This file seems quite random tbh.